web/src/Documentation.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from 'react'
+import {
+    useParams, Link
+} from "react-router-dom";
+
+import Markdown from 'react-markdown'
+import remarkGfm from 'remark-gfm'
+import remarkToc from 'remark-toc'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+// import { materialDark as syntaxStyle } from 'react-syntax-highlighter/dist/esm/styles/prism'
+
+interface IDocumentationArticleProps {
+    articleId?: string
+}
+
+function RouterLink(props: any) {
+    if (props.href.match(/^(https?:)?\/\//))
+        return <a href={props.href}>{props.children}</a>
+
+
+    let link: string = props.href
+    if (!link.startsWith('/')) { link = '/documentation/' + link; }
```

Comment:
Hey @danielreti, I'll reply to you in the inline comments :)
Just setting `link` to a variable here, as I want to rewrite if it's a relative link - as the relative links generated by the markdown don't work with this format.